src/ratings/dtos/create-rating.dto.ts

```diff
@@ -1,9 +1,11 @@
 import { PickType } from '@nestjs/swagger';
 import { Rating } from '../entities/rating.entity';
+import { IsNotEmpty } from 'class-validator';
 
 export class CreateRatingDto extends PickType(Rating, [
   'stars',
   'text',
 ] as const) {
+  @IsNotEmpty()
```

Comment:
I just found out that there is something called @IsUUID. Maybe try it out and maybe directly add it to every uuid or add a comment to #39